.vscode/settings.json

```diff
@@ -1,7 +1,14 @@
 {
   "tailwindCSS.experimental.configFile": {
-    "apps/storybook-react/tailwind.config.js": "packages/design-system-react/src/**",
-    "apps/storybook-react-native/tailwind-intellisense.config.js": "packages/design-system-react-native/src/**"
+    "apps/storybook-react/tailwind.config.js": [
+      "apps/storybook-react/src/**",
+      "packages/design-system-react/src/**",
+      "packages/design-tokens/stories/**"
+    ],
+    "apps/storybook-react-native/tailwind-intellisense.config.js": [
+      "packages/design-system-react-native/src/**",
+      "apps/storybook-react-native/stories/**"
+    ]
   },
```

Comment:
Move this to a separate PR to improve tailwind linting https://github.com/MetaMask/metamask-design-system/issues/653